base_business_document_import/models/business_document_import.py

```diff
@@ -179,7 +179,24 @@ def _match_partner(
     @api.model
     def _hook_match_partner(
             self, partner_dict, chatter_msg, domain, partner_type_label):
-        return False
+        # looking for the exact contact, we use email and contact name.
+        # if the email is a perfect hit we proceed with that, if not we use the
+        # partner name contained in all partners of our company
+        partner = False
+        if partner_dict.get('email'):
+            email = partner_dict['email'].replace(' ', '')
```

Comment:
do you mean `.strip()`?

Reply:
would be better, anyway emails never have spaces. will change